mesonbuild/dependencies/misc.py

```diff
@@ -253,7 +253,13 @@ def __init__(self, environment, kwargs):
         language = kwargs.get('language')
         super().__init__('openmp', environment, language, kwargs)
         self.is_found = False
-        openmp_date = self.compiler.get_define('_OPENMP', '', self.env, [], [self])
+        try:
+            openmp_date = self.compiler.get_define('_OPENMP', '', self.env, [], [self])
+        except mesonlib.EnvironmentException as e:
+            mlog.debug('OpenMP support not available in the compiler')
```

Comment:
We discussed this on IRC; no other dependency prints a warning when it's not found, and we don't have a consistent way to communicate the reason for a dependency not being found yet. Debug logging seemed simplest.

I like the other warning because it signals an invalid configuration rather than a compiler that just doesn't support openmp.